### ISSUE TITLE
terraform import myrasec_settings doesn't work

### ISSUE THREAD

**Issue by dshumsky** [Myra-Security-GmbH/terraform-provider-myrasec#24]
terraform import myrasec_settings.domain "anything"
returns
│ 
│ Error: Error fetching settings
│ 
│ EOF
│

**Comment by xellio** [comment]
We will fix this. I will inform you as soon as we release a new/fixed version of the terraform provider.

**Comment by xellio** [comment]
Hello @dshumsky  
I released a new version of the provider (v1.20.0). In this version, the import and usage of settings is fixed. You can import settings for a subdomain like

```
resource "myrasec_settings" "www_example_com" {
  subdomain_name = "www.example.com"
}
```


```
terraform import myrasec_settings.www_example_com www.example.com
```

Please update your provider version and let me know if it works for you.

**Comment by dshumsky** [comment]
it's working, Thank you @xellio
